geometry_msgs/Pose manipulator_pose
geometry_msgs/PoseArray bag_poses
bool setup_environment
string[] bag_labels
string[] bag_domain_labels
string pickup_object
string grasp_type
string drop_object
string planning_domain
bool safe_config
bool go_to_raised
---
bool success
